resources/js/components/Home/Footer.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <footer class="bg-white shadow">
+    <div class="w-full mx-auto max-w-screen-xl p-4 md:flex md:items-center md:justify-between">
+      <span class="text-sm text-gray-500 sm:text-center dark:text-gray-200">© Collegium Witelona</span>
```

Comment:
```suggestion
      <span class="text-sm text-gray-500 sm:text-center dark:text-gray-200">© Collegium Witelona Uczelnia Państwowa</span>
```

Reply:
Will change to `© Collegium Witelona: Testy`

Reply:
```suggestion
      <span class="text-sm text-gray-500 sm:text-center dark:text-gray-200">© Collegium Witelona: Testy</span>
```